test/language/expressions/generators/name-property-descriptor.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: >
+    Generator functions should define a `name` property.
+includes: [propertyHelper.js]
+es6id: 25.2.4
+---*/
+
+var g = function*() {};
+
+verifyNotEnumerable(g, 'name');
+verifyNotWritable(g, 'name');
+verifyConfigurable(g, 'name');
```

Comment:
Seems ok to be separate. But generally, a descriptor contains a value property, so it makes some sense to combine them - you're testing all the descriptor properties.